docs/faq/CodeStructure.md

```diff
@@ -72,3 +73,27 @@ Find the balance between these two extremes, and you will master Redux.
 - [#1165: Where to put business logic / validation?](https://github.com/reactjs/redux/issues/1165)
 - [#1171: Recommendations for best practices regarding action-creators, reducers, and selectors](https://github.com/reactjs/redux/issues/1171)
 - [Stack Overflow: Accessing Redux state in an action creator?](http://stackoverflow.com/questions/35667249/accessing-redux-state-in-an-action-creator/35674575)
+
+
+<a id="structure-action-creators"></a>
+### Why should I use action creators?
+
+Action creators are superior to inline actions for several reasons. 
```

Comment:
Let's rephrase this, and add a bit of extra info in here as well.

First, I'd say that action creators are not a required part of Redux, and you can dispatch actions directly from a component if you want, but they are a good practice for several reasons.  (I'd also remove the phrase "superior to inline actions".)

Would also be nice if we can squeeze in a mention that they aren't "just" an idea with Redux, but that Redux inherited the idea from the original Flux architecture, per the Flux docs at https://facebook.github.io/react/blog/2014/07/30/flux-actions-and-the-dispatcher.html#actions-and-actioncreators .